delphi/app/routes.py

```diff
@@ -1,23 +1,17 @@
 from app import app
 from flask import request
-import cv2
-import numpy
 
 
-@app.route('/')
-@app.route('/index')
-def index():
-    return 'Hello, World!'
+@app.route('/update-sensor-data', methods=['POST'])
+def updateSensorData():
+    data = request.get_json(silent=True)
```

Comment:
Silent param prevents html 404 from being thrown and instead returns None.

Reply:
shouldn't we return a 400 response if the data isn't there?

Reply:
We still do. This just prevents the 400 from being returned as html.